NDTensors/src/lib/Sectors/src/Sectors.jl

```diff
@@ -0,0 +1,19 @@
+module Sectors
+
+using ..SortedSets
+import ..SortedSets: AbstractSmallSet
+using InlineStrings
+using HalfIntegers
+
+import LinearAlgebra: norm
+import Base: *, ==, getindex, intersect, union, setdiff, symdiff, iterate, length, show
+#import NDTensors: ⊗
+#import NDTensors.SortedSets: AbstractSmallSet
+
+export ⊗, ⊕, U, SU, SUd, SUz, Z, Fib, Ising, Sector, nactive, show
```

Comment:
I have been going with the strategy of not exporting functions defined in submodule libraries at first and requiring end users to do `using NDTensors.Sectors: ⊗`, to be conservative at first. We can decide on exports later once things are more developed.

Reply:
Sounds good – let's make that change when appropriate.